app/javascript/components/collections/landing/Facets.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { makeStyles } from '@material-ui/core/styles';
```

Comment:
We should not use this `material-ui` convention (helper function) for applying styles to Avalon components.  `material-ui` is an indirect dependency brought in by Timeliner I'm guessing, but not listed in Avalon's `package.json` file.  If Timeliner goes away, this will throw an error.  

So if we want to keep the styles inline, then should handle something like:

```
const styles = {
   root: {
      width: "100%"
   }
   ...
}